src/env.h

```diff
@@ -255,6 +255,7 @@ class Environment {
     inline uint32_t* fields();
     inline int fields_count() const;
     inline bool call_init_hook();
+    bool enabled;
```

Comment:
I brought it up mostly because it looked incongruous.  I'll defer to @trevnorris here.